.github/workflows/release.yml

```diff
@@ -46,6 +46,17 @@ jobs:
         run: bash dev/ci-prepare.sh
       - name: "Make archives"
         run: python -u ./dev/releases/make_archives.py
+
+      - name: "Upload artifacts"
+        if: ${{ !startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Uhh, I didn't even check if this syntax is valid...

Reply:
The syntax should be correct according to GitHub Docs.